HomeAssistant/Views/SettingsDetailViewController.swift

```diff
@@ -788,6 +795,37 @@ enum AppIcon: String, CaseIterable {
     }
 }
 
+enum OpenInBrowser: CaseIterable {
```

Comment:
need a healthy dolop of `.rawValue` and `OpenInBrowser(rawValue: …)` around these UserDefaults hook points -- can't store the enum directly in, and `as? OpenInBrowser` sadly won't work either